quinn-h3/src/client.rs

```diff
@@ -86,14 +92,31 @@ pub struct Client {
 }
 
 impl Client {
-    pub fn connect(
+    pub fn connect(&self, url: &Url) -> Result<Connecting, ConnectError> {
```

Comment:
I'm a bit wary of adding this method, and I feel this doesn't add that much easy of use compared to the previous `connect()` (now `connect_socket()`) method.

Reply:
I got those changes removed.